mm2src/coins/eth.rs

```diff
@@ -2328,7 +2328,8 @@ impl MarketCoinOps for EthCoin {
                     Ok(conf) => {
                         if conf == confirmed_at {
                             status.append(" Confirmed.");
-                            break Ok(());
+                            // Let's avoid risking additional errors by calling `current_block` and instead return required_confirms
+                            break Ok(required_confirms.as_u64());
```

Comment:
It's already called above. Can't we utilize it here?

Reply:
Could you clarify what is called above?

To find number of confirmed blocks we need to get current block and subtract from it the confirmation_block_number.

I decided that we can return `required_confirms` value (instead of manual number of confirmations calculation, current block - tx confirmed at), as we rechecked that tx reached necessary confirmation block.
I mean, I think we could not to call `current_block`, bcz it potentially can return error at the very last moment of `wait_for_confirmations` function.

Reply:
My mistake. I confused `current_block` with `wait_for_block`.